internal/distribution/distroregistry_test.go

```diff
@@ -30,11 +30,14 @@ func TestDistroRegistry_List(t *testing.T) {
 		"rhel-93",
 		"rhel-94",
 		"rhel-95",
+		"rhel-9.6",
```

Comment:
Quick question out of curiosity. How come we change to a dot release here?

Reply:
Thanks. Ya, thought the change in composer happened a while back. Makes sense to drop the aliases... but obviously that can be done in a different PR at a later stage